code/modules/robotics/bot/firebot.dm

```diff
@@ -218,7 +234,7 @@
 				if(src.setup_party)
 					src.speak(pick("IT IS PARTY TIME.","I AM A FAN OF PARTIES", "PARTIES ARE THE FUTURE"))
 				else
-					src.speak(pick("I AM GOING TO MURDER THIS FIRE.","KILL ALL FIRES.","I DIDN'T START THIS, BUT I'M GOING TO END IT.","[TIME >= 30 MINUTES ? "TONIGHT" : "TODAY"] A FIRE DIES."))
```

Comment:
?

Reply:
Oh, forgot to document that. It was supposed to change the text after 30 minutes had passed in the round. I didnt realize TIME meant the server's time and not the in-game time, so it would only work for the first 30 minutes of the *day*! So I originally meant to use world.timeofday.